app/src/main/res/layout-hdpi/fragment_control_advanced.xml

```diff
@@ -132,35 +154,40 @@
                     android:padding="@dimen/control_margin_small">
 
                     <TextView
+                        style="@style/TextAppearance.AppCompat.Medium"
                         android:id="@+id/tv_control_advanced6"
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
-                        android:layout_weight="3"
+                        android:layout_weight="0.815"
```

Comment:
Hi @vikasmahato ! Have you tried the layouts in different screen sizes? Using emulator? This much fractions wouldn't be needed! :)